app/scripts/modules/amazon/src/instance/details/InstanceTags.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import _orderBy from 'lodash/orderBy';
```

Comment:
the rest of the deck does `import { orderBy } from 'lodash'`.  nbd, but nice to be consistent, IMO